src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/index.tsx

```diff
@@ -76,6 +77,7 @@ function _defaultComponents() {
 		h6: MdxH6,
 		p: MdxP,
 		table: MdxTable,
+		TryHcpCallout: MdxTryHcpCallout,
```

Comment:
Oh interesting I just realized the [cloud hcp callout ](https://github.com/search?q=repo%3Ahashicorp%2Fcloud.hashicorp.com+HCPCallout&type=code)isn't available locally maybe?

Reply:
Oh sorry that is only in the cloud repo so nvm I think.

Reply:
Yeah the name almost-collision got to me a lil 😬 I think we expose `HCPCallout` for HCP docs specifically here: https://github.com/hashicorp/dev-portal/blob/zs.try-hcp-callout-mdx/src/views/docs-view/index.tsx#L62